admin_site/viewsets.py

```diff
@@ -8,6 +8,7 @@
 from django.core.exceptions import FieldDoesNotExist
 from django.db.models import Model, QuerySet
 from django.forms import BaseModelForm
+from django.utils.translation import gettext_lazy
```

Comment:
In our code `gettext_lazy` is usually imported like this, maybe should follow the convention to be consistent:
```
from django.utils.translation import gettext_lazy as _
```